pkg/flare/archive.go

```diff
@@ -658,6 +677,7 @@ func walkConfigFilePaths(tempDir, hostname string, confSearchPaths SearchPaths,
 
 				if permsInfos != nil {
 					permsInfos.add(src)
+					addParentPerms(src, permsInfos)
```

Comment:
Same comment as for `addParentPerms(src, permsInfos)`.

Reply:
This one has to be inside at least the `Walk` because otherwise it may add paths where there are no configs `opt/etc/datadog-agent` on macos for example.